docs/user-guide/cli-using-team-profiles-hierarchy.md

```diff
@@ -0,0 +1,111 @@
+
+## How configuration files and profiles work together
+
+There may be instances where a user has all four types of files in their system, and all four configurations are referred to simultaneously by Zowe CLI for a particular command. **[is "simultaneously" correct?]**
+
+This can mean working with files that have contradictory configurations. One file can specify that a certain profile property has a value of `ABC`, while another file uses `XYZ` as a value.
+
+When the same properties have different values across multiple configuration files, Zowe CLI follows a two-step check to determine which configurations apply:
+
+1. Does the configuration file have a more *specialized* or more *generalized* use?
+2. Is the configuration file *more specific* or *less specific*.
+
+Zowe CLI considers a user configuration file to have a more specific use than a team configuration file, and a project configuration file to be more specialized than a global configuration file, which is more generalized.
+
+When checking all possible configuration file types, Zowe CLI categorizes files in the manner below:
+
+![Zowe CLI Config File Table](../images/common/cli-config-file-table.jpg)
+
+This order is applied no matter the directory in which you issue a Zowe CLI command. As a user, it can be easy to trace this logic when configuration files are all either in your home directory (i.e., specialized) or your project directory (i.e., generalized).
+
+But when there are configuration files across directories (meaning, in a project directory *and* a home directory), tracking how these files work together can seem more complicated.
+
+Read on to go over some examples.
+
+## Using a profile found in multiple configuration files
+
+Consider a user that has all configuration file types as in the following scenario:
+
+| specificity type | file type | profile | property | value |
+|----------- | ----------- | ----------- | ----------- | ----------- |
+| specialized/more specific | project user config file | One | ABC | red |
+| specialized/less specific | project team config file | Two | XYZ | yellow |
+| generalized/more specific | global user config file | Three | MNO | green |
+| generalized/less specific | global team config file | Two | XYZ | blue |
+
+In the case above, if Zowe CLI needs the `MNO` property to carry out a command, it refers to the global user configuration file to apply the `green` value because it is the only configuration file that has this particular property. No need to compare the specificity of files here.
+
+On the other hand, if a Zowe CLI command needs the information in the `Two` profile, it can seem like there are two possible values, `yellow` and `blue`. In this case Zowe CLI knows to use `yellow` by following the rules of specificity: The project team file is more specific than the global team file.
+
+Zowe CLI takes the following steps:
+
+1. Finds the `XYZ` property in both Two profiles.
```

Comment:
Should this say ``in both `Two` profiles`` or `in both profiles named "Two"` to make it clearer?

Reply:
Used the former. Thank you!